### PR TITLE
nodePackages.cdk8s-cli: init at 1.0.0-beta.5

### PR DESCRIPTION
**Pull request by ndarwincorn**
###### Motivation for this change
This adds [cdk8s-cli](https://github.com/awslabs/cdk8s) using generate.sh

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by midchildan — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
@ndarwincorn can you please rebase this? Hit me up on IRC to let me merge it before it conflicts again.

**Comment by ndarwincorn** [comment]
@SuperSandro2000 - I've rebased this. I'll ping you on IRC

File: pkgs/development/node-packages/composition.nix [hunk 1]
```diff
@@ -6,12 +6,12 @@
 
 let
   nodeEnv = import ./node-env.nix {
-    inherit (pkgs) stdenv python2 util-linux runCommand writeTextFile;
+    inherit (pkgs) stdenv python2 utillinux runCommand writeTextFile;
```
Comment (SuperSandro2000) [inline pkgs/development/node-packages/composition.nix:9]: Until we update node2nix you need to revert the utillinux changes. Sorry about that.

Comment (ndarwincorn) [reply pkgs/development/node-packages/composition.nix:9]: no worries, node2nix automatically made that change when I ran generate.sh and I was curious whether I should've included it

**Comment by SuperSandro2000** [comment]
I am probably fixing the merge conflict that will be here shortly in another PR and closing this one,

**Comment by SuperSandro2000** [comment]
PR got combined with a few others other node updates. Closing so that the update goes trough smoothly.

**Pull request closed without merging**